src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx

```diff
@@ -37,30 +49,51 @@ export const ItemFlow = ({ name, index, onRemove }: ItemFlowProps) => {
   const { error } = getFieldState(itemName);
 
   const title = (
-    <StyledTitle component="span" sx={{ position: 'relative' }}>
```

Comment:
maybe position: relative is needed here, for example for error badge?
could you pls check?

Reply:
I just moved it to StyledTitle in ItemFlow.styles.ts